core/src/main/java/org/kiwix/kiwixmobile/core/downloader/downloadManager/FetchDownloadNotificationManager.kt

```diff
@@ -108,6 +108,20 @@ class FetchDownloadNotificationManager @Inject constructor(
     }
   }
 
+  override fun getSubtitleText(
+    context: Context,
+    downloadNotification: DownloadNotification
+  ): String {
+    return when {
+      downloadNotification.isCompleted -> context.getString(R.string.fetch_notification_download_complete)
+      downloadNotification.isFailed -> context.getString(R.string.fetch_notification_download_failed)
+      downloadNotification.isPaused -> context.getString(R.string.fetch_notification_download_paused)
+      downloadNotification.isQueued -> context.getString(R.string.fetch_notification_download_resuming)
+      downloadNotification.etaInMilliSeconds < 0 -> context.getString(R.string.fetch_notification_download_downloading)
+      else -> super.getSubtitleText(context, downloadNotification)
+    }
+  }
+
```

Comment:
After renaming the string name, please use our string in the `getCancelNotification` method of the `FetchNotificationManager` class so that it will use our text instead of fetch.